tests/test_admon/test_plugin_parse.yml

```diff
@@ -0,0 +1,145 @@
+- attrs:
+  - - class
+    - admonition note
+  block: true
+  children: null
+  content: Note
+  hidden: false
+  info: ' note'
+  level: 0
+  map:
+  - 0
+  - 2
+  markup: ???
+  meta:
+    tag: note
+  nesting: 1
+  tag: div
+  type: admonition_open
+- attrs:
+  - - class
+    - admonition-title
+  block: true
+  children: null
+  content: ''
+  hidden: false
+  info: ''
+  level: 1
+  map:
+  - 0
+  - 1
+  markup: ??? note
```

Comment:
Good point on adding a test for the markup. I added the snippet: https://github.com/executablebooks/mdit-py-plugins/pull/58#discussion_r1111225521

Let me know if there is anything else!

Reply:
Thanks @KyleKing 
I think though, to align with code fences, etc, it should be:

```yaml
  info: note
  markup: ??? 
```

does that make sense?

Reply:
I'm entirely sure what should be where. I went ahead and created a diff between how the currently released version of `mdit` would parse the files vs. the changes here:

https://github.com/executablebooks/mdit-py-plugins/pull/58/commits/0258a7f6a1a76aac5418be435d7c821234bed455

The plugin does extract `info: ???` and `markup: note` like you mentioned above, for at least one section, but not all, which I think is expected?

Reply:
Just following up. Is there anything I can do to help merge this PR? Or any resources or comparisons that I can look at for determining what the output should be? Thanks for all of your feedback so far!